x-pack/plugins/cases/public/components/case_view/index.tsx

```diff
@@ -359,9 +361,10 @@ export const CaseComponent = React.memo<CaseComponentProps>(
             title={caseData.title}
           >
             <CaseActionBar
-              currentExternalIncident={currentExternalIncident}
               caseData={caseData}
+              currentExternalIncident={currentExternalIncident}
               disabled={!userCanCrud}
+              isAlerting={ruleDetailsNavigation != null}
```

Comment:
Why don't we pass a `disableAlerts` variable as you did for the `AllCasesGeneric` component?

Reply:
cuz there is already a disabled prop in the `CaseActionBar`. We will have showAlertDetails default to true but we're not going to show the action if `ruleDetailsNavigation` is not passed